### ISSUE TITLE
main view UX

### ISSUE THREAD

**Issue by gdaunlam** [yy0931/sqlite3-editor#26]
I opened the issue to share my experience while managing a main view. Initially, I used the query editor view as the main view, but I found myself limited as I couldn't sort or filter based on values in more than one column from the UI. I addressed my needs by using custom queries, but I encountered other issues along the way, which were resolved during the week, thanks to the excellent support provided to the plugin.

The only friction I encounter today is the inability to add or delete rows from a table using the UI of a custom query. Perhaps if the add and delete views had a table selector, it would suffice, although it's a minor issue because it can be resolved by switching back and forth to the query editor.

In summary, I am inclined to stop using the query editor view altogether, as it represents more of a dependency than a tool.

**Comment by thanhle7** [comment]
Have tried all the sqlite tools on vscode market place at the moment and found this one is the best.

**Comment by gdaunlam** [comment]
Yes, it is the best in the market, it is excellent, and it will surely be so for a long time. 
My only idea is to raise the things I believe could be improved, and I imagine that other users may have some ideas they want to contribute.
Maybe it will be useful to @yy0931  or maybe not, and in that case, I will close the issue.

**Comment by yy0931** [comment]
Sorry for my inactivity in this thread. I thought implementing it was not urgent, but I will do when I have time.
Edit: And thanks for your kind words, thanhle7 and gdaunlam!

**Comment by gdaunlam** [comment]
Okay, thank you, no problem, and the response is appreciated!

**Comment by yy0931** [comment]
I've added support for record deletion to custom queries in version 1.0.146. You can use that feature by dragging your cursor over the row numbers. The query must reference either the rowid or a primary key column.
